### PR TITLE
Add link to /community-moderation on all /mod pages

### LINKED ISSUE #4369: Add link to /community-moderation on all /mod pages
It's difficult to find links to the community moderation guide ([dev.to/community-moderation](https://dev.to/community-moderation)), so it'd be valuable to add this link to various relevant pages.

We should add this link - https://dev.to/community-moderation to any `/mod` page: 
1. [https://dev.to/mod](https://dev.to/mod)
2. https://dev.to/dev-username/article-title/mod

**Additional context**

1. Suggestion on where to put link on `/mod`:

image

2. Suggestion on where to put link on `/dev-username/article-title/mod`:

image

### PR DESCRIPTION
**Pull request by sabunt**
## What type of PR is this? (check all applicable)
resolve #4369 
- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Add links for all mod pages
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screenshot 2019-10-24 at 02 00 11
Screenshot 2019-10-24 at 02 44 47

Screenshot 2019-10-24 at 02 44 19
Screenshot 2019-10-24 at 02 44 37

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by sabunt** [comment]
@rhymes plz review

File: app/views/moderations/index.html.erb [hunk 1]
```diff
@@ -108,11 +109,11 @@
             <% end %>
           </center>
           <div class="body">
-            <%= HTML_Truncator.truncate(
+            <%= raw HTML_Truncator.truncate(
```
Comment (Zhao-Andy) [inline app/views/moderations/index.html.erb]: Hmm any particular reason for this? I'm reading some stuff now and it seems to be a performance improvement, but want to hear your thoughts on the change.

File: app/views/moderations/index.html.erb [hunk 2]
```diff
@@ -85,6 +85,7 @@
   <br /><br /><br /><br />
   <center>
     <h1>Mod Dashboard (beta)</h1>
+    <a href="/community-moderation" style="display: inline-block; font-size: 1.3em; padding: 3px 8px; border-radius: 3px;">Community moderation guide</a>
```
Comment (Zhao-Andy) [inline app/views/moderations/index.html.erb]: I think we can move these styles into a new `.scss` file, maybe `.moderation.scss` or something.

File: app/views/moderations/mod.html.erb [hunk 3]
```diff
@@ -122,6 +122,7 @@
   <% end %>
   <% is_mod = (@moderatable.tag_list.select { |t| current_user.has_role?(:tag_moderator, Tag.find_by(name: t)) }).any? if @moderatable.class.name == "Article" %>
   <% if @moderatable.class.name == "Article" && (current_user.has_role?(:super_admin) || is_mod) %>
+    <a href="/community-moderation" style="display: inline-block; font-size: 1.3em; padding: 3px 8px; border-radius: 3px;">Community moderation guide</a>
```
Comment (Zhao-Andy) [inline app/views/moderations/mod.html.erb]: Same as above

File: app/views/moderations/index.html.erb (hunk 1 above)
Comment (sabunt) [reply app/views/moderations/index.html.erb]: First of all, the one subtle difference between raw and html_safe: raw(nil) returns an empty string, while nil.html_safe throws an exception, secondary it's easy for reading.

**Comment by rhymes** [comment]
Hi @sabunt, I've merged master to solve a conflict, hope you don't mind! :)

**Pull request merged**